cookbooks/bcpc-hadoop/recipes/datanode.rb

```diff
@@ -182,6 +202,10 @@
   Chef::Application.fatal!("You have fewer #{node[:bcpc][:hadoop][:disks]} than #{node[:bcpc][:hadoop][:hdfs][:failed_volumes_tolerated]}! See comments of HDFS-4442.")
 end
 
+link "/etc/init.d/hadoop-yarn-nodemanager" do
```

Comment:
This could have been at line 122, where the previous template for yarn init script was.